File: app/src/main/java/br/com/disapps/homepet/ui/map/IMapView.java
package br.com.disapps.homepet.ui.map;

import br.com.disapps.homepet.data.model.Coordinate;
import br.com.disapps.homepet.ui.common.AppView;

/**
 * Created by diefferson.santos on 23/08/17.
 */

public interface IMapView  extends AppView {

    void fillCoordinates(Coordinate coordinate);
}

File: app/src/main/java/br/com/disapps/homepet/ui/map/IMapView.kt
package br.com.disapps.homepet.ui.map

import br.com.disapps.homepet.data.model.Coordinate
import br.com.disapps.homepet.ui.common.AppView

/**
* Created by diefferson.santos on 23/08/17.
**/
interface IMapView : AppView {

    fun fillCoordinates(coordinate: Coordinate)
}
